src/frontend/src/flows/addDevice/manage/addDeviceSuccess.test.ts

```diff
@@ -0,0 +1,61 @@
+import { renderAddDeviceSuccess } from "./addDeviceSuccess";
+import copyJson from "./addDeviceSuccess.json";
+
+describe("addDeviceSuccess", () => {
+  const deviceAlias = "Test device alias";
+
+  const { en } = copyJson;
+
+  const waitFor = (callback: () => void): Promise<void> =>
+    new Promise((resolve) => {
+      const start = Date.now();
+
+      const callCallback = () => {
+        if (Date.now() > start + 10 * 1000) {
+          resolve();
+          return;
+        }
+
+        try {
+          callback();
+          resolve();
+        } catch (err: unknown) {
+          // Ignore error and try again in a bit
+          setTimeout(callCallback, 100);
+        }
+      };
+
+      callCallback();
+    });
```

Comment:
this shouldn't not be necessary, the templates should be rendered immediately or after one or two ticks, depending on the logic (in this case one tick)

Reply:
it's actually mandatory because I use `renderAddDeviceSuccess` without awaiting in the test. The test also must not await otherwise it cannot go through since the function `resolve` only on click.

I plan to extract a test utils in another PR.